### PR TITLE
add python/sanic server

### PR DESCRIPTION
**Pull request by sharkspeed**
add a python server : sanic

### CODE REVIEW

**Comment by tbrand** [comment]
Awesome, I'll try this! Thanks!

File: python/sanic/server_python_sanic.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3.5
```
Comment (tbrand) [inline python/sanic/server_python_sanic.py]: Is the python 3.x required for sanic?

File: python/sanic/server_python_sanic.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+# -*- coding: utf-8 -*-
+
+from sanic import Sanic
```
Comment (tbrand) [inline python/sanic/server_python_sanic.py]: When I try this, I got following error.
May be you should put requirement.txt at the directory and install sanic in Makefile?
```
ImportError: No module named 'sanic'
```

File: python/sanic/server_python_sanic.py (hunk 1 above)
Comment (sharkspeed) [inline python/sanic/server_python_sanic.py]: yeah. Python 3.5+ is required.

File: python/sanic/server_python_sanic.py (hunk 2 above)
Comment (sharkspeed) [inline python/sanic/server_python_sanic.py]: Ok, I will add it.

**Comment by tbrand** [comment]
Great! It works!!:tada:

**Comment by tbrand** [comment]
As I mentioned here https://github.com/tbrand/which_is_the_fastest/pull/34, is there any way to disable all loggings?

**Comment by tbrand** [comment]
Sorry I've reverted this PR.

**Pull request merged**